libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -206,15 +300,36 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+
+        // Blending error (weight of sensor in blending)
+        float blending_factor;
```

Comment:
just like the cal, could we make a mini struct called blend that contains healthy and factor

Reply:
That's a left over